pkg/config/policies.go

```diff
@@ -53,8 +53,8 @@ func (c *ConfigTx) GetPoliciesForConsortiums() (map[string]Policy, error) {
 	return getPolicies(consortiums.Policies)
 }
 
-// GetPoliciesForConsortium returns a map of policies for a specific consortium.
-func (c *ConfigTx) GetPoliciesForConsortium(consortiumName string) (map[string]Policy, error) {
+// ConsortiumPolicies returns a map of policies for a specific consortium.
+func (c *ConfigTx) ConsortiumPolicies(consortiumName string) (map[string]Policy, error) {
```

Comment:
This and ConsortiumsPolicies can be removed now that FAB-17665 is merged

Reply:
🛠